src/components/toggle/Toggle.js

```diff
@@ -53,15 +53,15 @@ const Toggle = React.forwardRef<Props, HTMLInputElement>(
         const classes = classNames('toggle-container', className, {
             'is-toggle-right-aligned': isToggleRightAligned,
         });
-        let toggleElements = [
+
+        const toggleElements = [
             <div key="toggle-simple-switch" className="toggle-simple-switch" />,
             <div key="toggle-simple-label" className="toggle-simple-label">
                 {label}
             </div>,
         ];
-
         if (isToggleRightAligned) {
-            toggleElements = toggleElements.reverse();
```

Comment:
this reassignment doesn't do anything because `reverse()` mutates the original array so it's assigning it to itself: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/reverse#the_reverse_method_returns_the_reference_to_the_same_array